### PR TITLE
Update Troubadore; Versatile Ranger.json

### PR DESCRIPTION
**Pull request by NDLP95**
Hello, this is an updated version of my homebrew. An older version was originally submitted to 5eTools by another person.

### CODE REVIEW

File: class/Troubadore; Versatile Ranger.json [hunk 1]
```diff
@@ -3,13 +3,13 @@
...
 			{
 				"json": "TheVersatileRanger",
-				"abbreviation": "TTVR",
+				"abbreviation": "TVR",
 				"full": "Troubadore's Versatile Ranger",
 				"authors": [
 					"Troubadore"
 				],
 				"convertedBy": [
-					"Matsozetex#8230"
```
Comment (lorddusk) [inline class/Troubadore; Versatile Ranger.json:12]: Leave this name in, they also converted it. And this tag is supposed to be a full list of people who worked on it

Comment (NDLP95) [reply class/Troubadore; Versatile Ranger.json:12]: Ah gotcha, I didn't know! Er, so sorry, but I'm super new to github, so may I ask how to do this? Should I go back to my fork and edit it, making a new PR?

Comment (lorddusk) [reply class/Troubadore; Versatile Ranger.json:12]: I've added it for you. So it's all good now.

Comment (NDLP95) [reply class/Troubadore; Versatile Ranger.json:12]: Thanks so much!

**Pull request merged**